src/ui/public/vislib/lib/data.js

```diff
@@ -1,10 +1,11 @@
 import d3 from 'd3';
 import _ from 'lodash';
-import VislibComponentsZeroInjectionInjectZerosProvider from '../components/zero_injection/inject_zeros';
-import VislibComponentsZeroInjectionOrderedXKeysProvider from '../components/zero_injection/ordered_x_keys';
-import VislibComponentsLabelsLabelsProvider from '../components/labels/labels';
+import { VislibComponentsZeroInjectionInjectZerosProvider } from '../components/zero_injection/inject_zeros';
+import { VislibComponentsZeroInjectionOrderedXKeysProvider } from '../components/zero_injection/ordered_x_keys';
+import { VislibComponentsLabelsLabelsProvider } from '../components/labels/labels';
 import VislibComponentsColorColorProvider from 'ui/vis/components/color/color';
```

Comment:
Initially because I was waiting for this PR to get merged: https://github.com/elastic/kibana/pull/11251

But now it's merged so I'll convert.